helpers/asset.js

```diff
@@ -15,6 +22,8 @@ module.exports = function (type, filename) {
     out = '<link rel="stylesheet" href="'+filePath+'"/>';
   } else if (type == 'js') {
     out = '<script src="'+filePath+'"></script>';
+  } else if (type == 'image') {
+    out = filePath;
```

Comment:
Could, but I was thinking of a use case like `style="background: url({{asset "image" "someimage.jpg"}});"`